components/src/core/components/Input/PasswordInput.vue

```diff
@@ -1,16 +1,141 @@
 <template>
-  <oxd-input type="password" />
+  <div :class="strengthClass">
+    <oxd-text class="password-strength-check" v-if="showStrength">{{
+      strengthLabel
+    }}</oxd-text>
+    <oxd-input
+      :type="isPasswordVisible ? 'text' : 'password'"
```

Comment:
Format using [style guide](https://vuejs.org/style-guide/) pay attention to attribute order

```html
<oxd-input
   v-bind="$attrs"
   class="password-input"
   :hasError="hasError"
   :disabled="disabled"
   :readonly="readonly"
   :type="isPasswordVisible ? 'text' : 'password'"
></oxd-input>
```

Reply:
done